radis/lbl/calc.py

```diff
@@ -28,17 +28,331 @@
+                           Trot=Trot,
+                           pressure=pressure,
+                           overpopulation=overpopulation,
+                           molecule=molecule,
+                           isotope=isotope,
+                           mole_fraction=mole_fraction,
+                           path_length=1,
+                           databank=databank,
+                           medium=medium,
+                           wstep=wstep,
+                           broadening_max_width=broadening_max_width, 
+                           lineshape_optimization=lineshape_optimization,
+                           name=name,
+                           use_cached=use_cached,
+                           verbose=verbose,
+                           **kwargs)
+        return s
+    
+    else: # The "molecules" dictionary is read, instead of the possible other inputs ("molecule", "mole_fraction", ...)    
+        s = []
+        
+        # Check consistency of inputs
+        for mol in molecule:
+            try:
+                assert mol in isotope.keys()
+                assert mol in overpopulation.keys()
+                assert mol in mole_fraction.keys()
+                assert mol in databank.keys()
+            except AssertionError:
+                raise ValueError('mole_fraction, overpopulation, databank and isotope '+\
+                                 'should be dictionaries of listing all molecules')
+            
+            
+            s.append(_calc_spectrum(wavenum_min=wavenum_min,
+                                   wavenum_max=wavenum_max,
+                                   wavelength_min=wavelength_min,
+                                   wavelength_max=wavelength_max,
+                                   Tgas=Tgas,
+                                   Tvib=Tvib,
+                                   Trot=Trot,
+                                   pressure=pressure,
+                                   overpopulation=overpopulation[mol],
+                                   molecule=mol,
+                                   isotope=isotope[mol],
+                                   mole_fraction=mole_fraction[mol],
+                                   path_length=path_length,
+                                   databank=databank[mol],
+                                   medium=medium,
+                                   wstep=wstep,
+                                   broadening_max_width=broadening_max_width, 
+                                   lineshape_optimization=lineshape_optimization,
+                                   name=name,
+                                   use_cached=use_cached,
+                                   verbose=verbose,
+                                   **kwargs))
+        if len(s) == 1:
+            return s[0]
+        else:
+            from radis import MergeSlabs        
+            s_tot = MergeSlabs(s)
+            return s_tot
 
-def calc_spectrum(
+def _calc_spectrum(
```

Comment:
In this one I would: 

- remove all the defaults (no = 1, "DLM", "fetch", anything but None). We use the defaults from calc_spectrum() anyway. 
- remove the docstring entirely.   Maybe replace it with ```See :py:func:`~radis.lbl.calc.calc_spectrum` ````